app/views/portal/index.rb

```diff
@@ -185,6 +185,26 @@ def facets_selected
       end
     end
 
+    def federated_search_enabled
+      @collection && @collection.federation_configs.count.positive?
+    end
+
+    def federated_search_conf
+      mustache[:federated_search_conf] ||= begin
+        {
+          tab_items: @collection.federation_configs.map do |config|
+            foederati_provider = Foederati::Providers.get(config.provider)
+            next unless foederati_provider
```

Comment:
Instead of this, you could use:
```ruby
@collection.federation_configs.select { |config| Foederati::Providers.get(config.provider).present? }.map do |config|
```
... so the `next` line can be removed, as can the `#reject` call at the block's end.